Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM node:18-alpine
+FROM node:18-slim
```

Comment:
I am unsure I understand why `slim` is needed, but not a big deal

Reply:
Very good point.  Let me show:

- Failed [apline](https://github.com/PatrickFuhrmann-HTWBerlin/oai-provider-service/actions/runs/10059807904/job/27805867258)
- Succeeded [slim](https://github.com/PatrickFuhrmann-HTWBerlin/oai-provider-service/actions/runs/10059887570/job/27806127477)

Reply:
Looks like alpine does not  have the arm emulation  support needed

Reply:
That's what I thought, but 'docker manifest inspect node:18-slim' and 'docker manifest inspect node:18-apline' are essentially showing the same supported architectures.